cloudinit/distros/arch.py

```diff
@@ -62,56 +58,6 @@ def install_packages(self, pkglist: PackageList):
         self.update_package_sources()
         self.package_command("", pkgs=pkglist)
 
-    def _get_renderer(self) -> Renderer:
```

Comment:
The base class still has this method, do you think we still need it?

Reply:
Yes, this is still used by `apply_network_config`